src/index.html

```diff
@@ -17,6 +17,90 @@
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <div class="header__container">
+        <div class="header__logo">
+          <a
+            href="#moyo"
+            class="header__logo-link"
+          >
+            <img
+              src="images/logo.png"
+              alt="moyo logo"
```

Comment:
The `alt` attribute for the image tag should describe the image content. In this case, it's better to describe what is on the logo, for example, 'Moyo company logo'. This will improve your page SEO and helps screen readers.